packages/account-v2/src/containers/ManualForm/ManualFormDocumentUpload.tsx

```diff
@@ -24,9 +24,9 @@ export const ManualFormDocumentUpload = ({ selectedDocument }: TManualFormDocume
                         <FormDocumentUploadField
                             buttonText='Drop file or click here to upload'
                             className='h-full'
-                            description={upload.text}
+                            description={upload?.text}
                             fileFormats={['image/jpeg', 'image/jpg', 'image/png', 'image/gif', 'application/pdf']}
-                            icon={upload.icon}
+                            icon={upload?.icon}
                             maxSize={8388608}
                             name={upload.pageType}
```

Comment:
shouldnt we add. {upload?.pageType} as well?

Reply:
pageType will always be there in the config. icon and title will be optional for selfieWithID

Reply:
If upload can't be undefined, we don't need optional chaining.
We only `upload?.` incase the upload is undefined.